### ISSUE TITLE
Not working for Laravel 4.2

### ISSUE THREAD

**Issue by syphernl** [restricted/authchain#1]
This installs fine on Laravel 4.2 but when being used it throws an exception:

```
Class Restricted\Authchain\ChainAuthenticationProvider contains 2 abstract methods and must therefore be declared abstract or implement the remaining methods (Illuminate\Auth\UserProviderInterface::retrieveByToken, Illuminate\Auth\UserProviderInterface::updateRememberToken) 
```

**Comment by restricted** [comment]
Seems to be updated UserProviderInterface, working on fix now.

**Comment by restricted** [comment]
Fixed in versions 1.0.6 and above.
